index.js

```diff
@@ -344,7 +344,12 @@ URL.prototype.toString = function toString(stringify) {
   if (url.hash) result += url.hash;
 
   return result;
-};
+}
+if (Object.defineProperty) {
+  Object.defineProperty(URL.prototype, 'toString', { value: toString });
```

Comment:
`writable`, `enumerable`, and `configurable` should be `true` in the descriptor.